nuxeo-core/nuxeo-core-bulk/src/test/java/org/nuxeo/ecm/core/bulk/CoreBulkFeature.java

```diff
@@ -47,25 +46,16 @@
 public class CoreBulkFeature implements RunnerFeature {
 
     public static class DummyLogin implements LoginAs {
-
         @Override
         public LoginContext loginAs(String username) throws LoginException {
             return Framework.login();
         }
     }
 
-    public static class BulkWaiter implements Waiter {
-        @Override
-        public boolean await(Duration duration) throws InterruptedException {
-            BulkService bulks = Framework.getService(BulkService.class);
-            return bulks.await(duration);
-        }
-
-    }
-
     @Override
     public void initialize(FeaturesRunner runner) {
-        runner.getFeature(TransactionalFeature.class).addWaiter(new BulkWaiter());
+        runner.getFeature(TransactionalFeature.class)
+              .addWaiter(deadline -> Framework.getService(BulkService.class).await(Duration.ofMinutes(1)));
```

Comment:
You're changing logic there, duration becomes deadline which is no more taken into account.